docs/advanced_topics/customisation/page_editing_interface.md

```diff
@@ -115,11 +115,11 @@ The unique key used to identify the format. To unregister this format, call `unr
 **`label`**\
 The label used in the chooser form when inserting the image into the `RichTextField`.
 
-**`classnames`**\
+**`classname`**\
 The string to assign to the `class` attribute of the generated `<img>` tag.
```

Comment:
These docs are really only for the args, as these are not kwargs the naming here does not matter but I thought it would be good to update nonetheless.

I have intentionally not included a changed in 5.2 note as I thought this would just cause more confusion.